CMakeLists.txt

```diff
@@ -162,7 +162,7 @@ endif(USE_CUDA)
 if(USE_CUDA)
   cuda_add_library(dgl SHARED ${DGL_SRC})
 else(USE_CUDA)
-  add_library(dgl SHARED ${DGL_SRC})
+  add_library(dgl SHARED ${DGL_SRC} src/graph/sampling/randomwalks/node2vec_impl_tmp.h)
```

Comment:
Did you include this file in the PR?

Reply:
It is a temporary file. I forgot to delete in the Cmakelist